backend/app/Http/Controllers/StudentController.php

```diff
@@ -27,13 +28,8 @@ public function store(Request $request) {
       'password' => bcrypt($fields['password'])
     ]);
 
-    $token = $student->createToken('myapptoken')->plainTextToken;
+    $student->createToken('myapptoken')->plainTextToken;
```

Comment:
It this function still needed? If not, I think we can remove it

Reply:
Ok Sir, I think we can remove that one. I'll update this right now.